cmd/pulumi-test-language/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/pulumi/pulumi/cmd/pulumi-test-language
 
-go 1.21
+go 1.22
+
+toolchain go1.22.1
```

Comment:
Is the `toolchain` bit required here?

Reply:
`make tidy` added it, so I didn't question it tbh. Though if I remove it `make tidy` doesn't add it back, so I think it can be removed.